### PR TITLE
Modeling users passwords

### PR DESCRIPTION
**Pull request by adamerdemer**
Added in secure passwords

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	let(:user){ described_class.new(name: "Sam", email: "test@test.com", password: "12345", password_confirmation: "12345") }
+
+  describe "setup" do
+  	it "creates a valid user" do
+  		expect(user.valid?).to be(true)
+  	end
+  	it "checks user has a name attribute" do
+  		expect(user.name).to eq("Sam")
+  	end
+		it "checks user has a email attribute" do
```
Comment (dbugsy) [inline spec/models/user_spec.rb:13]: Indentation needs to be fixed

**Review by dbugsy — CHANGES_REQUESTED** [review]
I'm interested in why you are building this manually - devise gem would automate this. Design decisions should be added to the PR to avoid questions like this.

**Comment by dbugsy** [comment]
Please move trello ticket into the correct column @adamerdemer @Robert-G-J

**Comment by adamerdemer** [comment]
@ddosque I used this method because the link on the ticket said to not use devise in your first rails project. https://github.com/plataformatec/devise#starting-with-rails

We followed the instructions via Micheal Hartl's book.

**Comment by sblausten** [comment]
:shipit:

**Comment by Robert-G-J** [comment]
@adamerdemer @danielemanuel Is this pull request still valid? Or has it been overwritten by your work on Devise?

**Pull request closed without merging**